articles/openshift/oshift-how-apply-security-best-practices.md

```diff
@@ -0,0 +1,70 @@
+---
+title: How to apply security best practices in OpenShift
+description: Apply security best practices in OpenShift
```

Comment:
Change to something like:
Provides guidance on security best practices to ensure that your OpenShift cluster to minimise the attack vector to a UKCloud OpenShift cluster and its workloads